pnpm-lock.yaml

```diff
@@ -17228,8 +17228,8 @@ packages:
     dependencies:
       '@aws-crypto/sha256-browser': 3.0.0
       '@aws-crypto/sha256-js': 3.0.0
-      '@aws-sdk/client-sso-oidc': 3.569.0(@aws-sdk/client-sts@3.569.0)
-      '@aws-sdk/client-sts': 3.569.0
+      '@aws-sdk/client-sso-oidc': 3.569.0
+      '@aws-sdk/client-sts': 3.569.0(@aws-sdk/client-sso-oidc@3.569.0)
```

Comment:
Is there any way to get rid of the aws stuff? All from oclif? Would stop me from noting in just about every update that these are shifted and probably shouldn't be.
@CraigMacomber + @Josmithr, please check the lock diffs for oddities. 
(This set of oddities are very common from Tyler originated updates.)

Reply:
I did see that change. I considered it fine as its slightly different transative dep versioning of packages which are trasnative deps of the thing being updated. This kind of change is common when multiple packaegs depend on the same dep through diferent paths and doing a version update.

Really does weem like a lot more deps that we should need for running CLI tools though.

Reply:
One option we could consider is overriding the deps completely to get them out of the lockfile: https://github.com/microsoft/FluidFramework/pull/21979

For posterity, this is one of the AWS oclif features that we don't use: https://oclif.io/docs/releasing/#uploading-to-s3 This and features like it are why the aws deps are included in oclif.